sdk/servicebus/azure-servicebus/swagger/servicebus-swagger.json

```diff
@@ -0,0 +1,943 @@
+              "xml": {
+                "attribute": true
+              }
+            },
+            "TopicDescription": {
+              "description": "Topic information to create.",
+              "$ref": "#/definitions/TopicDescription"
+            }
+          }
+        }
+      }
+    },
+    "EntityAvailabilityStatus": {
+      "description": "Availibility status of the entity",
+      "type": "string",
+      "xml": {
+        "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+      },
+      "enum": [
+        "Available",
+        "Limited",
+        "Renaming",
+        "Restoring",
+        "Unknown"
+      ]
+    },
+    "EntityStatus": {
+      "description": "Status of a Service Bus resource",
+      "type": "string",
+      "xml": {
+        "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+      },
+      "enum": [
+        "Active",
+        "Creating",
+        "Deleting",
+        "Disabled",
+        "ReceiveDisabled",
+        "Renaming",
+        "Restoring",
+        "SendDisabled",
+        "Unknown"
+      ]
+    },
+    "MessageCountDetails": {
+      "description": "Details about the message counts in queue.",
+      "type": "object",
+      "properties": {
+        "ActiveMessageCount": {
+          "description": "Number of active messages in the queue, topic, or subscription.",
+          "type": "integer",
+          "xml": {
+            "prefix": "d2p1",
+            "namespace": "http://schemas.microsoft.com/netservices/2011/06/servicebus"
+          }
+        },
+        "DeadLetterMessageCount": {
+          "description": "Number of messages that are dead lettered.",
+          "type": "integer",
+          "xml": {
+            "prefix": "d2p1",
+            "namespace": "http://schemas.microsoft.com/netservices/2011/06/servicebus"
+          }
+        },
+        "ScheduledMessageCount": {
+          "description": "Number of scheduled messages.",
+          "type": "integer",
+          "xml": {
+            "prefix": "d2p1",
+            "namespace": "http://schemas.microsoft.com/netservices/2011/06/servicebus"
+          }
+        },
+        "TransferDeadLetterMessageCount": {
+          "description": "Number of messages transferred into dead letters.",
+          "type": "integer",
+          "xml": {
+            "prefix": "d2p1",
+            "namespace": "http://schemas.microsoft.com/netservices/2011/06/servicebus"
+          }
+        },
+        "TransferMessageCount": {
+          "description": "Number of messages transferred to another queue, topic, or subscription.",
+          "type": "integer",
+          "xml": {
+            "prefix": "d2p1",
+            "namespace": "http://schemas.microsoft.com/netservices/2011/06/servicebus"
+          }
+        }
+      },
+      "xml": {
+        "name": "CountDetails",
+        "namespace": "http://schemas.microsoft.com/netservices/2011/06/servicebus/connect"
```

Comment:
Should be http://schemas.microsoft.com/netservices/2010/10/servicebus/connect

Reply:
Good catch. Updated.